model/inline/src/main/scala/aqua/model/inline/ArrowInliner.scala

```diff
@@ -366,7 +368,7 @@ object ArrowInliner extends Logging {
       _ <- Arrows[S].purge
 
       abilityResolvingResult <- abArgs.toList.traverse { case (str, (vm, sct)) =>
```

Comment:
Lets define `Monoid` for `AbilityResolvingResult` and add `.map(_.fold)` after `traverse` here?